timev.rb

```diff
@@ -274,24 +284,55 @@ def self.at(time, subsec = false, unit = :microsecond, in: nil)
     end
   end
 
-  # Returns a new \Time object based on the given arguments.
+  # Returns a new \Time object based on the given arguments,
+  # by default in the local timezone.
   #
   # With no positional arguments, returns the value of Time.now:
   #
-  #   Time.new                                       # => 2021-04-24 17:27:46.0512465 -0500
+  #   Time.new # => 2021-04-24 17:27:46.0512465 -0500
+  #
+  # With one to six arguments, returns a new \Time object
+  # based on the given arguments, in the local timezone.
+  #
+  #   Time.new(2000, 1, 2, 3, 4, 5) # => 2000-01-02 03:04:05 -0600
+  #
+  # There are no minimum and maximum values for +year+;
+  # otherwise, these are the minimum and maximum values:
+  #
+  #   Time.new(-2000, 1, 1, 0, 0, 0)
+  #   # => -2000-01-01 00:00:00 -0600
+  #   Time.new(2000, 12, 31, 23, 59, 60)
+  #   # => 2001-01-01 00:00:00 -0600 # Note the rollover cause by 60 sec.
+  #
+  # The values may be numerics convertible to integers:
+  #
+  #   Time.new(Float(0.0), Rational(1, 1), 2, 3, 4, 5)
+  #   # => 0000-01-02 03:04:05 -0600
+  #
+  # Or string integers:
+  #
+  #   a = %w[0 1 2 3 4 5]
+  #   # => ["0", "1", "2", "3", "4", "5"]
+  #   Time.new(*a) # => 0000-01-02 03:04:05 -0600
   #
-  # Otherwise, returns a new \Time object based on the given parameters:
+  # And the value for +month+ may be a case-insensitive 3-letter month name:
   #
-  #   Time.new(2000)                                 # => 2000-01-01 00:00:00 -0600
-  #   Time.new(2000, 12, 31, 23, 59, 59.5)           # => 2000-12-31 23:59:59.5 -0600
-  #   Time.new(2000, 12, 31, 23, 59, 59.5, '+09:00') # => 2000-12-31 23:59:59.5 +0900
+  #   a = %w[0 jan 2 3 4 5] # => ["0", "jan", "2", "3", "4", "5"]
+  #   Time.new(*a) # => 0000-01-02 03:04:05 -0600
+  #   a = %w[0 JAN 2 3 4 5] # => ["0", "JAN", "2", "3", "4", "5"]
+  #   Time.new(*a) # => 0000-01-02 03:04:05 -0600
```

Comment:
Similarly, we can simplify these examples:

```suggestion
  #   Time.new(2000, "jan") # => 2000-01-01 00:00:00 -0500
  #   Time.new(2000, "JAN") # => 2000-01-01 00:00:00 -0500
```

Reply:
Done.